### PR TITLE
Introduce a more-minimal base theme.

### PR DESCRIPTION
**Pull request by gwax**
This is a somewhat more austere _base_ theme, meant to provide details for the discussion in getnikola/nikola#2744

### CODE REVIEW

File: v7/base-austere/templates/base_header.tmpl [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            %endif
+        % endif
+    %endfor
+    ${template_hooks['menu']()}
+    ${template_hooks['menu_alt']()}
+    </ul>
+    </nav>
+</%def>
+
+<%def name="html_translation_header()">
```
Comment (Kwpolska) [inline v7/base-austere/templates/base_header.tmpl]: What’s wrong with keeping it in base_helper.tmpl under the old name?

File: v7/base-austere/templates/base_footer.tmpl [hunk 2]
```diff
@@ -0,0 +1,10 @@
+## -*- coding: utf-8 -*-
```
Comment (Kwpolska) [inline v7/base-austere/templates/base_footer.tmpl]: What did you change here? Avoid duplicating templates if you only delete the namespace (which I agree is unnecessary, perhaps we should to that upstream)

File: v7/base-austere/templates/feed_helper.tmpl [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+<%def name="head(name=None)">
+    <%
+        if name is None:
+            atom_ref = 'index_atom'
+            rss_ref = 'rss'
+            label = ''
+        else:
+            atom_ref = kind + '_atom'
+            rss_ref = kind + '_rss'
+            label = ' for ' + kind + ' ' + name
```
Comment (Kwpolska) [inline v7/base-austere/templates/feed_helper.tmpl]: Hm. I’d really like to see something like this upstream, but with i18n support, optional JSONFeed support, and without a raw Python block (which are not allowed in Jinja2) — changes to the core codebase to allow this everywhere would be more welcome.

File: v7/base-austere/templates/index_helper.tmpl [hunk 4]
```diff
@@ -0,0 +1,19 @@
+## -*- coding: utf-8 -*-
```
Comment (Kwpolska) [inline v7/base-austere/templates/index_helper.tmpl]: I would keep that mathjax_script helper and not change this template. Perhaps some people want to switch from `base` to `base-austere`, and they will get unusual errors about missing things.

File: v7/base-austere/templates/page.tmpl [hunk 5]
```diff
@@ -0,0 +1 @@
+<%inherit file="story.tmpl"/>
```
Comment (Kwpolska) [inline v7/base-austere/templates/page.tmpl]: Delete this file due to no changes.

File: v7/base-austere/assets/css/rst.css [hunk 6]
```diff
@@ -0,0 +1,330 @@
+/*
```
Comment (Kwpolska) [inline v7/base-austere/assets/css/rst.css]: No need to duplicate this file if you inherit from base.

File: v7/base-austere/templates/post.tmpl [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+    ${helper.html_pager(post)}
+    </nav>
+    </aside>
+    % if not post.meta('nocomments') and site_has_comments:
+        <section class="comments hidden-print">
+        <h2>${messages("Comments")}</h2>
+        ${comments.comment_form(post.permalink(absolute=True), post.title(), post._base_path)}
+        </section>
+    % endif
+    ${math.math_scripts_ifpost(post)}
```
Comment (Kwpolska) [inline v7/base-austere/templates/post.tmpl]: I had to make a diff and I found out we had a bug in base (scripts vs styles). I fixed that in getnikola/nikola@d9ee04f4, but it’d be great if you would also make changes to the built-in base theme.

(That file is unnecessary after that fix.)

**Comment by Kwpolska** [comment]
In `rst.css`:

```diff
/* reset inner alignment in figures */
-.figure.align-right {
+div.align-right {
   text-align: inherit }
```

Why?

File: v7/base-austere/templates/post_helper.tmpl [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+    %elif 'site' in twitter_card:
+    <meta name="twitter:site" content="${twitter_card['site']}">
+    %endif
+    %if 'creator:id' in twitter_card:
+    <meta name="twitter:creator:id" content="${twitter_card['creator:id']}">
+    %elif 'creator' in twitter_card:
+    <meta name="twitter:creator" content="${twitter_card['creator']}">
+    %endif
+%endif
+</%def>
```
Comment (Kwpolska) [inline v7/base-austere/templates/post_helper.tmpl]: Restore `mathjax_script` as well.

**Comment by gwax** [comment]
Maybe I'll try to cut a PR against upstream for the structural changes and then see what's left.

**Comment by Kwpolska** [comment]
What’s the status of this? Do we really need this?

**Comment by gwax** [comment]
I think enough stuff got pared down in my rework of the base theme that this can be closed out.

**Pull request closed without merging**